### ISSUE TITLE
Improve colour contrast for visited links in Dark Theme

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2003]
Gray text on a black background isn't great:

<img width="820" alt="Screen Shot 2021-03-20 at 12 14 38 PM" src="https://user-images.githubusercontent.com/427398/111876637-ffc1c500-8975-11eb-9abf-8bbc9a31e7a7.png">

This has an [AA rating vs. AAA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast), and I bet we could do better.
